homeassistant/components/zwave_js/binary_sensor.py

```diff
@@ -0,0 +1,389 @@
+    },
+    {
+        # NotificationType 3: Carbon Dioxide - All other State Id's
+        "type": NOTIFICATION_CARBON_DIOXIDE,
+        "states": [4, 5, 7],
+        "device_class": DEVICE_CLASS_GAS,
+        "enabled": False,
+    },
+    {
+        # NotificationType 4: Heat - State Id's 1, 2, 5, 6 (heat/underheat)
+        "type": NOTIFICATION_HEAT,
+        "states": [1, 2, 5, 6],
+        "device_class": DEVICE_CLASS_HEAT,
+    },
+    {
+        # NotificationType 4: Heat - All other State Id's
+        "type": NOTIFICATION_HEAT,
+        "states": [3, 4, 8, 10, 11],
+        "device_class": DEVICE_CLASS_HEAT,
+        "enabled": False,
+    },
+    {
+        # NotificationType 5: Water - State Id's 1, 2, 3, 4
+        "type": NOTIFICATION_WATER,
+        "states": [1, 2, 3, 4],
+        "device_class": DEVICE_CLASS_MOISTURE,
+    },
+    {
+        # NotificationType 5: Water - All other State Id's
+        "type": NOTIFICATION_WATER,
+        "states": [5],
+        "device_class": DEVICE_CLASS_MOISTURE,
+        "enabled": False,
+    },
+    {
+        # NotificationType 6: Access Control - State Id's 1, 2, 3, 4 (Lock)
+        "type": NOTIFICATION_ACCESS_CONTROL,
+        "states": [1, 2, 3, 4],
+        "device_class": DEVICE_CLASS_LOCK,
+    },
+    {
+        # NotificationType 6: Access Control - State Id 22 (door/window open)
+        "type": NOTIFICATION_ACCESS_CONTROL,
+        "states": [22],
+        "device_class": DEVICE_CLASS_DOOR,
+    },
+    {
+        # NotificationType 7: Home Security - State Id's 1, 2 (intrusion)
+        # Assuming that value 1 and 2 are not present at the same time
+        "type": NOTIFICATION_HOME_SECURITY,
+        "states": [1, 2],
+        "device_class": DEVICE_CLASS_SAFETY,
+    },
+    {
+        # NotificationType 7: Home Security - State Id's 3, 4, 9 (tampering)
+        "type": NOTIFICATION_HOME_SECURITY,
+        "states": [3, 4, 9],
+        "device_class": DEVICE_CLASS_SAFETY,
+    },
+    {
+        # NotificationType 7: Home Security - State Id's 5, 6 (glass breakage)
+        # Assuming that value 5 and 6 are not present at the same time
+        "type": NOTIFICATION_HOME_SECURITY,
+        "states": [5, 6],
+        "device_class": DEVICE_CLASS_SAFETY,
+    },
+    {
+        # NotificationType 7: Home Security - State Id's 7, 8 (motion)
+        "type": NOTIFICATION_HOME_SECURITY,
+        "states": [7, 8],
+        "device_class": DEVICE_CLASS_MOTION,
+    },
+    {
+        # NotificationType 8: Power management - Values 1...9
+        "type": NOTIFICATION_POWER_MANAGEMENT,
+        "states": [1, 2, 3, 4, 5, 6, 7, 8, 9],
+        "device_class": DEVICE_CLASS_POWER,
+        "enabled": False,
+    },
+    {
+        # NotificationType 8: Power management - Values 10...15
+        # Battery values (mutually exclusive)
+        "type": NOTIFICATION_POWER_MANAGEMENT,
+        "states": [10, 11, 12, 13, 14, 15],
+        "device_class": DEVICE_CLASS_POWER,
```

Comment:
We have a device class for binary battery sensors:
https://github.com/home-assistant/core/blob/4e71be852af8b42a3f43c8da79df4bb956439a80/homeassistant/components/binary_sensor/__init__.py#L25-L26